osu.Desktop/Program.cs

```diff
@@ -168,12 +168,30 @@ private static bool trySendIPCMessage(IIpcHost host, string cwd, string[] args)
 
         private static void setupVelopack()
         {
-            VelopackApp
-                .Build()
-                .WithFirstRun(v =>
+            var app = VelopackApp.Build();
+
+            app.WithFirstRun(_ =>
+            {
+                if (OperatingSystem.IsWindows())
+                    WindowsAssociationManager.InstallAssociations();
+            });
+
+            if (OperatingSystem.IsWindows())
```

Comment:
what is this for?

Reply:
Reciting the OP:
`WithAfterUpdateFastCallback` is marked as only available on windows, so it requires an OS check to be able to run without warnings. `WindowsAssociationManager.InstallAssociations()` is also marked as only available for windows, and since it's in a different scope, it also requires an OS check of its own.

Reply:
Have fixed this.